tracer/src/Datadog.Trace/ClrProfiler/NativeCallTargetDefinition3.cs

```diff
@@ -1,41 +1,42 @@
-// <copyright file="NativeCallTargetDefinition2.cs" company="Datadog">
+// <copyright file="NativeCallTargetDefinition3.cs" company="Datadog">
```

Comment:
3?

Reply:
Yeah, if we're not sending these from managed -> Native, I'm not sure why we need this file at all? 🤔